File: Sem- 1/C/Array/duplicate.c
#include<stdio.h>
void main(){
	printf("No. of Duplicate Elements");
	int n,i,temp;
	printf("\nEnter no. of Elements: ");
	scanf("%d",&n);
	int arr[n],j;
	for(i=0;i<n;i++){
		printf("Enter Number(%d): ",i+1);
		scanf("%d",&arr[i]);
	}
	printf("Array: ");
	for(i=0;i<n;i++){
		printf("%d ",arr[i]);
	}
	for(i=0;i<n;i++){
		for(j=i+1;j<n;j++){
			if(arr[i]>arr[j]){
			temp=arr[i];
			arr[i]=arr[j];
			arr[j]=temp;
		    }
		}
	}
	int count=0,k=0,total=0;
	printf("\n");
	while(k<n){
		if(arr[k]==arr[k+1]){
			count++;
			k++;
		}
		else{
			printf("No. of Duplicates of %d: %d\n",arr[k],count);
			total+=count;
			k++;
			count=0;
		}
	}
	printf("Total No. of Duplicates in Array: %d",total);
}

File: Sem- 3/Data Structure/Array/Duplicate.java
import java.util.Scanner;
import java.util.Arrays;
public class Duplicate{
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		System.out.println("Duplicate Elements in Array");
		System.out.print("Enter no. of elements in array: ");
		int n=sc.nextInt();
		int[] arr=new int[n];
		for(int i=0;i<arr.length;i++){
			System.out.print("Enter value at arr["+i+"]: ");
			arr[i]=sc.nextInt();
		}
		System.out.print("Array: ");
		for(int j=0;j<arr.length;j++){
			System.out.print(arr[j]+" ");
		}
		Arrays.sort(arr);
		System.out.print("\nDuplicates: ");
		int count=0,k=0;
		for(int i=0;i<arr.length-1;i++){
			if(arr[i]==arr[i+1]){
				count++;
				System.out.print(arr[i]+" ");
			}	
		}
		if(count==0){
			System.out.print("None");
		}
		System.out.println("\nNo. of Duplicates: "+count);
	}
}